### PR TITLE
Rename Risk Metrics Terminology to Student Metrics

### PR DESCRIPTION
**Pull request by victorhuangwq**
## Description

We have decided the following terminologies to:
at-risk students → students in need
risk-metrics → student metrics 

This PR only covers the UI of the metrics feature. (Excludes the documentation)

## Motivation and Context

Based on discussion, we have realized that at risk may carry negative connotations. 

## How Has This Been Tested?

image
image
image
image
image
image
image
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: app/views/assessments/index.html.erb [hunk 1]
```diff
@@ -66,7 +66,7 @@
           <% end %>
 
           <% if @cud.instructor? %>
-           <%= link_to "<div class='row metrics'><i class='material-icons left'>assessment</i> Risk Metrics</div>".html_safe, course_metrics_path, title: "View student metrics", :class => "btn btn-large red darken-3" %>
+           <%= link_to "<div class='row metrics'><i class='material-icons left'>assessment</i> Student Metrics</div>".html_safe, course_metrics_path, title: "View Metrics for Identifying Students in Need", :class => "btn btn-large red darken-3" %>
```
Comment (amartinez116) [inline app/views/assessments/index.html.erb:69]: The wording sounds a bit off. Maybe change it to just "Metrics for Identifying Students in Need"

File: app/assets/javascripts/watchlist.js [hunk 2]
```diff
@@ -236,16 +236,16 @@ function get_watchlist_function(){
         });
         
         pending_html = `<div class="ui secondary segment" >
-                     <h5> Pending at-risk students </h5>
+                     <h5> Pending students in need </h5>
```
Comment (amartinez116) [inline app/assets/javascripts/watchlist.js:239]: "Pending students in need of help"?

File: app/assets/javascripts/watchlist.js [hunk 3]
```diff
@@ -236,16 +236,16 @@ function get_watchlist_function(){
...
                     </div>`;
         contacted_html = `<div class="ui secondary segment" >
-                          <h5> Contacted at-risk students </h5>
+                          <h5> Contacted students in need </h5>
```
Comment (amartinez116) [inline app/assets/javascripts/watchlist.js:242]: The rest can just be "x students" and no "x students in need"

**Review by amartinez116 — CHANGES_REQUESTED** [review]

**Comment by victorhuangwq** [comment]
image
image
image

**Comment by victorhuangwq** [comment]
I have made the appropriate changes. Lmk if this looks good to you

**Review by amartinez116 — APPROVED** [review]
LGTM. We can further discuss wording if you want to change it

**Pull request merged**